stable/elastalert/templates/deployment.yaml

```diff
@@ -54,8 +54,24 @@ spec:
 {{- end }}
         resources:
 {{ toYaml .Values.resources | indent 12 }}
-{{- if .Values.optEnv }}
         env:
+{{- if .Values.elasticsearch.existingSecret }}
```

Comment:
Maybe call this `credentialsSecret` or something?

Reply:
Done